packages/components/src/core/Dropdown/index.tsx

```diff
@@ -128,12 +110,27 @@ const Dropdown = <
     );
   }
 
+  const isMultiColumn = options && !!options[0] && "options" in options[0];
```

Comment:
discussion: Maybe this? `const isMultiColumn = "options" in (options?.[0] || EMPTY_OBJECT);`

Reply:
Hmm, let me try this 😯